win/WIN_UIelement.cpp

```diff
@@ -45,7 +44,6 @@ void UIelement::setRect(const gfx::Rectangle& rect)
 	rect_ = rect;
 }
 
-
 void UIelement::mouseEnter()
 {
 	std::cout << "The mouse has entered the " << name_ << " UI element. \n";
```

Comment:
Use preprocessor here as we don't want these messages most of the time. Eg `#if defined(_DEBUG)` or `#if defined(VERBOSE)` where VERBOSE is a flag set in the project settings dialog.
**SHOULD**

Reply:
done